Content.Server/Chemistry/ReactionEffects/AreaReactionEffect.cs

```diff
@@ -78,7 +78,7 @@ public class AreaReactionEffect : IReactionEffect
         /// </summary>
         private string _sound;
 
-        public AreaReactionEffect()
+        protected AreaReactionEffect()
```

Comment:
why do this?

Reply:
idk the IDE suggested it. Makes sense though because AreaReactionEffect is an abstract class so there's no point in having a public constructor.